content/reports/1000_IBM Power - Virtual Machines/Host Power HMC.yaml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Thanks for adding the Hosts report! Suggest renaming this so that it is nested under an "IBM Power/Hosts" dir (instead of putting it with VMs):
```
rename:    1000_IBM Power - Virtual Machines/Host Power HMC.yaml -> 1000_IBM Power - Hosts/010_HMC Hosts.yaml
```

Reply:
Thanks Jay for the quick review. I modified this !